tests/scripts/ncp_mode

```diff
@@ -195,7 +196,7 @@ setup_infraif()
 {
     if ! ip link show backbone1 >/dev/null 2>&1; then
         echo "Creating backbone1 with Docker..."
-        docker network create --driver bridge --ipv6 --subnet 9101::/64 -o "com.docker.network.bridge.name"="backbone1" backbone1
+        docker network create --driver bridge -o "com.docker.network.bridge.name"="backbone1" backbone1
```

Comment:
I'm wondering about the purpose of having `9101::/64` in the old code. Don't we need it any more?

Reply:
As I recall, it was added because otherwise will get an error when specifying '--ipv6'. I just found without '--ipv6' all the functionality can still work. 

But with the subnet '9101::/.64', when doling the multicast ping, source address with this prefix will be preferred instead of the on-link address. And the ping fails. So I remove it here.

Reply:
Thanks! I also think we should prefer the on-link prefix over `9101::/64` in tests.